### ISSUE TITLE
 🚀 [Feature] Full Text Search

### ISSUE THREAD

**Issue by Tirokk** [Ocelot-Social-Community/Ocelot-Social#308]
> [<img alt="datenbrei" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2019-03-27T08:05:42Z" title="Wednesday, March 27th 2019, 9:05:42 am +01:00">Mar 27, 2019</time>_

---

### Full Text Search
![image](https://user-images.githubusercontent.com/1238238/55473417-fd41f580-560e-11e9-8d00-90efe73823f8.png)

- [ ] Functionality needs  2 be rechecked
- [x] Design has to be rechecked since there seems to be a different approach (result in dropdown?)
- [ ] Filter the Feed
- [ ] Must work with the filter feature https://github.com/Human-Connection/Human-Connection/issues/374
- [ ] Mobile Responsiveness
- [ ] Search for Authors
- [ ] "Search by tag" option #1068

### Full Text Search - Userstory

```gherkin
Feature: Search
  As a user
  I want to search all posts with a certain keyword
  To find a particular kind of content in case I know some keywords in advance
```

### Design & Layout - Alpha

![image](https://user-images.githubusercontent.com/1238238/55473417-fd41f580-560e-11e9-8d00-90efe73823f8.png)

![image](https://user-images.githubusercontent.com/1238238/55473443-0a5ee480-560f-11e9-81a9-d40342c679c4.png)

### Design & Layout - Nitro

![image](https://user-images.githubusercontent.com/1238238/55473478-1ea2e180-560f-11e9-887d-0c0c9302ec06.png)

![image](https://user-images.githubusercontent.com/1238238/55473505-2a8ea380-560f-11e9-95e1-177081b2cc51.png)

![image](https://user-images.githubusercontent.com/1238238/55473524-367a6580-560f-11e9-91aa-d7badc33469e.png)

**Comment by Tirokk** [comment]
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-04-11T09:08:10Z" title="Thursday, April 11th 2019, 11:08:10 am +02:00">Apr 11, 2019</time>_

---


# Disfunction

- [ ] If you search `fficia` – part of content word `officia` – it's not found. But the whole word `officia` is found.
- [ ] Search word is in the search field: If you click in the search field or mark the text, the text disappears.

## Circumstance

- FireFox, macOS

Sometimes it works correct and than disfunction is suddenly there again …
Could not find out, when it happens to change.

**Comment by Tirokk** [comment]
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-11T12:41:57Z" title="Tuesday, June 11th 2019, 2:41:57 pm +02:00">Jun 11, 2019</time>_

---


here are some screen shots of the functionality
![Screenshot at 2019-06-11 09:35:20](https://user-images.githubusercontent.com/26943915/59272393-54c29a00-8c2c-11e9-890d-d0dbbed8ca5a.png)
![Screenshot at 2019-06-11 09:35:07](https://user-images.githubusercontent.com/26943915/59272399-5724f400-8c2c-11e9-8744-ec1b6d7dc10b.png)
![Screenshot at 2019-06-11 09:34:42](https://user-images.githubusercontent.com/26943915/59272414-5f7d2f00-8c2c-11e9-8a5f-09d768df778b.png)
![Screenshot at 2019-06-11 09:32:54](https://user-images.githubusercontent.com/26943915/59272417-5f7d2f00-8c2c-11e9-98a4-9dd6ec77945d.png)
![Screenshot at 2019-06-11 09:32:31](https://user-images.githubusercontent.com/26943915/59272418-6015c580-8c2c-11e9-95ab-ffab59cc2792.png)

@Tirokk the fuzzy search seems to be working(?) It even takes into account there might be a misspelling on the part of the user.
Notice it doesn't clear after the user goes to the post, but it is clear-able with the check icon on the right of the search input
I guess maybe this is still open because of the filter functionality(?)
One thing I'll add to the check list :point_up: is mobile responsiveness 
![Screenshot at 2019-06-11 09:40:38](https://user-images.githubusercontent.com/26943915/59272784-25f8f380-8c2d-11e9-861e-5c75c4d3dff0.png)

**Comment by Tirokk** [comment]
> [<img alt="stale[bot]" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/apps/stale) **Authored by [stale[bot]](https://github.com/apps/stale)**
_<time datetime="2019-11-19T20:10:07Z" title="Tuesday, November 19th 2019, 9:10:07 pm +01:00">Nov 19, 2019</time>_

---


This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

**Comment by Tirokk** [comment]
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-12-12T07:06:26Z" title="Thursday, December 12th 2019, 8:06:26 am +01:00">Dec 12, 2019</time>_

---


I think it is still a point and @DennisHack action shows this as well.
So it should be not in stale mode …

**Comment by Tirokk** [comment]
> [<img alt="stale[bot]" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/apps/stale) **Authored by [stale[bot]](https://github.com/apps/stale)**
_<time datetime="2020-02-10T07:14:16Z" title="Monday, February 10th 2020, 8:14:16 am +01:00">Feb 10, 2020</time>_

---


This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

**Comment by Tirokk** [comment]
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-03-06T07:48:32Z" title="Friday, March 6th 2020, 8:48:32 am +01:00">Mar 6, 2020</time>_

---


Still for the future …

**Comment by Tirokk** [comment]
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-03-24T12:48:01Z" title="Tuesday, March 24th 2020, 1:48:01 pm +01:00">Mar 24, 2020</time>_

---


Is this still relevant?

**Comment by Tirokk** [comment]
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-23T06:56:19Z" title="Thursday, April 23rd 2020, 8:56:19 am +02:00">Apr 23, 2020</time>_

---


You know best @Mogge .
I think it is irrelevant if you ask this way.
Can you check the details for things still relevant or good ideas. And if you think „all“ relevant things are done in your actual PR #3271 then close this one.

**Comment by Tirokk** [comment]
> [<img alt="stale[bot]" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/apps/stale) **Authored by [stale[bot]](https://github.com/apps/stale)**
_<time datetime="2020-06-22T07:52:19Z" title="Monday, June 22nd 2020, 9:52:19 am +02:00">Jun 22, 2020</time>_

---


This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

**Comment by Tirokk** [comment]
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-22T08:20:54Z" title="Monday, June 22nd 2020, 10:20:54 am +02:00">Jun 22, 2020</time>_

---


Is this solved by PR #3271 as well @ogerly @Mogge ?

**Comment by Tirokk** [comment]
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-09-08T08:20:39Z" title="Tuesday, September 8th 2020, 10:20:39 am +02:00">Sep 8, 2020</time>_

---


yes is fixed #2144
